Mlem/App/Environment Values/ParentFrameWidth.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ParentFrameWidth.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-07-28.
+//
+
+import MlemMiddleware
+import SwiftUI
+
+struct ParentFrameWidth: EnvironmentKey {
+    static let defaultValue: CGFloat = .zero
+}
+
+extension EnvironmentValues {
+    var parentFrameWidth: CGFloat {
+        get { self[ParentFrameWidth.self] }
+        set { self[ParentFrameWidth.self] = newValue }
+    }
+}
```

Comment:
OOS but these days [there's a cleaner way](https://developer.apple.com/documentation/swiftui/entry()) of doing these definitions

Reply:
Wow, that's way better. IDK why all the guides I've found do it the hard way lol